### PR TITLE
COG-510 Cognito - Add acceptance tests for login screen

### PR DESCRIPTION
**Pull request by Nasir2712**
#### Which User story does this relate to (link)?
- https://osi-cwds.atlassian.net/browse/COG-510

#### Brief feature description
- Since the MFA code tests have already been written, I wrote acceptance tests for forgot password page scenarios.

### CODE REVIEW

File: spec/acceptance/support/login_helper.rb [hunk 1]
```diff
@@ -35,6 +35,17 @@ def cognito_login
...
   end
 
+  def click_forgot_password_link
+    visit ENV.fetch('RAILS_RELATIVE_URL_ROOT', '/')
+    return unless current_url.include?('login')
+    click_link 'Forgot your password?'
+  end
+
+  def reset_password
+    fill_in 'Email', with: ENV.fetch('COGNITO_USERNAME', 'y_test111+role1@outlook.com')
```
Comment (kelfink) [inline spec/acceptance/support/login_helper.rb]: not sure we should have a default email address that looks real.  Or is this real?

**Comment by Nasir2712** [comment]
sure

File: spec/acceptance/support/login_helper.rb (hunk 1 above)
Comment (Nasir2712) [reply spec/acceptance/support/login_helper.rb]: if you have any automated email that works when email submit, i'll be happy to use that. The ones that we are using don't work for reset password.

**Comment by Nasir2712** [comment]
I'll add those test cases as well. I'll move this to WIP.

**Pull request merged**